ccl/backupccl: TestSerializesScheduledBackupExecutionArgs failed

[(ccl/backupccl).TestSerializesScheduledBackupExecutionArgs failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2688496&tab=buildLog) on [master@3c223f5f5162103110a790743b687ef2bf952489](https://github.com/cockroachdb/cockroach/commits/3c223f5f5162103110a790743b687ef2bf952489):

```
      /go/src/github.com/cockroachdb/cockroach/pkg/sql/plan_opt.go:508 +0xbf7
  github.com/cockroachdb/cockroach/pkg/sql.(*planner).makeOptimizerPlan()
      /go/src/github.com/cockroachdb/cockroach/pkg/sql/plan_opt.go:194 +0x13d
  github.com/cockroachdb/cockroach/pkg/sql.(*connExecutor).makeExecPlan()
      /go/src/github.com/cockroachdb/cockroach/pkg/sql/conn_executor_exec.go:932 +0x6f
  github.com/cockroachdb/cockroach/pkg/sql.(*connExecutor).dispatchToExecutionEngine()
      /go/src/github.com/cockroachdb/cockroach/pkg/sql/conn_executor_exec.go:815 +0x1f8
  github.com/cockroachdb/cockroach/pkg/sql.(*connExecutor).execStmtInOpenState()
      /go/src/github.com/cockroachdb/cockroach/pkg/sql/conn_executor_exec.go:653 +0x14cd
  github.com/cockroachdb/cockroach/pkg/sql.(*connExecutor).execStmt()
      /go/src/github.com/cockroachdb/cockroach/pkg/sql/conn_executor_exec.go:121 +0xf53
  github.com/cockroachdb/cockroach/pkg/sql.(*connExecutor).execCmd.func1()
      /go/src/github.com/cockroachdb/cockroach/pkg/sql/conn_executor.go:1475 +0x3a7
  github.com/cockroachdb/cockroach/pkg/sql.(*connExecutor).execCmd()
      /go/src/github.com/cockroachdb/cockroach/pkg/sql/conn_executor.go:1477 +0x198c
  github.com/cockroachdb/cockroach/pkg/sql.(*connExecutor).run()
      /go/src/github.com/cockroachdb/cockroach/pkg/sql/conn_executor.go:1402 +0x40c
  github.com/cockroachdb/cockroach/pkg/sql.(*InternalExecutor).initConnEx.func1()
      /go/src/github.com/cockroachdb/cockroach/pkg/sql/internal.go:202 +0x8e

Goroutine 654 (running) created at:
  testing.(*T).Run()
      /usr/local/go/src/testing/testing.go:1168 +0x5bb
  github.com/cockroachdb/cockroach/pkg/ccl/backupccl.TestSerializesScheduledBackupExecutionArgs()
      /go/src/github.com/cockroachdb/cockroach/pkg/ccl/backupccl/create_scheduled_backup_test.go:381 +0x112e
  github.com/cockroachdb/cockroach/pkg/ccl/backupccl.TestSerializesScheduledBackupExecutionArgs()
      /go/src/github.com/cockroachdb/cockroach/pkg/ccl/backupccl/create_scheduled_backup_test.go:185 +0x124
  testing.tRunner()
      /usr/local/go/src/testing/testing.go:1123 +0x202

Goroutine 1205 (running) created at:
  github.com/cockroachdb/cockroach/pkg/sql.(*InternalExecutor).initConnEx()
      /go/src/github.com/cockroachdb/cockroach/pkg/sql/internal.go:201 +0x4a5
  github.com/cockroachdb/cockroach/pkg/sql.(*InternalExecutor).execInternal()
      /go/src/github.com/cockroachdb/cockroach/pkg/sql/internal.go:708 +0x971
  github.com/cockroachdb/cockroach/pkg/sql.(*InternalExecutor).QueryIteratorEx()
      /go/src/github.com/cockroachdb/cockroach/pkg/sql/internal.go:542 +0x2b5
  github.com/cockroachdb/cockroach/pkg/sql.(*InternalExecutor).QueryIterator()
      /go/src/github.com/cockroachdb/cockroach/pkg/sql/internal.go:528 +0x45
  github.com/cockroachdb/cockroach/pkg/sql/catalog/lease.(*Manager).DeleteOrphanedLeases.func1.1()
      /go/src/github.com/cockroachdb/cockroach/pkg/sql/catalog/lease/lease.go:2086 +0x121
  github.com/cockroachdb/cockroach/pkg/util/retry.WithMaxAttempts()
      /go/src/github.com/cockroachdb/cockroach/pkg/util/retry/retry.go:172 +0x14f
  github.com/cockroachdb/cockroach/pkg/sql/catalog/lease.(*Manager).DeleteOrphanedLeases.func1()
      /go/src/github.com/cockroachdb/cockroach/pkg/sql/catalog/lease/lease.go:2085 +0x304
  github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask.func1()
      /go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:351 +0x149
==================
    testing.go:1038: race detected during execution of test
    --- FAIL: TestSerializesScheduledBackupExecutionArgs/full-cluster-free_user (0.18s)

```

<details><summary>More</summary><p>
Parameters:

- TAGS=
- GOFLAGS=-race -parallel=4

```
make stressrace TESTS=TestSerializesScheduledBackupExecutionArgs PKG=./pkg/ccl/backupccl TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestSerializesScheduledBackupExecutionArgs.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

--- a/pkg/ccl/backupccl/create_scheduled_backup_test.go
+++ b/pkg/ccl/backupccl/create_scheduled_backup_test.go
@@ -33,7 +33,6 @@ import (
 	"github.com/cockroachdb/cockroach/pkg/sql/sessiondata"
 	"github.com/cockroachdb/cockroach/pkg/testutils"
 	"github.com/cockroachdb/cockroach/pkg/testutils/serverutils"
-	"github.com/cockroachdb/cockroach/pkg/testutils/skip"
 	"github.com/cockroachdb/cockroach/pkg/testutils/sqlutils"
 	"github.com/cockroachdb/cockroach/pkg/util/leaktest"
 	"github.com/cockroachdb/cockroach/pkg/util/log"
@@ -180,7 +179,6 @@ func (t userType) String() string {
 // itself with the actual scheduling and the execution of those backups.
 func TestSerializesScheduledBackupExecutionArgs(t *testing.T) {
 	defer leaktest.AfterTest(t)()
-	skip.UnderRaceWithIssue(t, 60718, "flaky test")
 	defer log.Scope(t).Close(t)
 
 	th, cleanup := newTestHelper(t)

